packages/components/constraints/src/horizontal/horizontal.tsx

```diff
@@ -1,11 +1,40 @@
 import React from 'react';
-import PropTypes from 'prop-types';
```

Comment:
We should remove this dep from `package.json` (I probably forgot to do it in the other packages I migrated as well).

Reply:
okay, will do :=